pkgs/by-name/fl/flawz/package.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+, sqlite
+, installShellFiles
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "flawz";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "orhun";
+    repo = "${pname}";
+    rev = "v${version}";
+    hash = "sha256-rd+Qwvn0+33Tc4GnahFqUwxkQncus03DMrd8rV7XbGY=";
+  };
+
+  cargoHash = "sha256-2LUM2POHcn149MkYC6qergkhepA0PI16TRYzFo5BhbY=";
+
+  nativeBuildInputs = [ pkg-config installShellFiles ];
+
+  buildInputs = [ openssl sqlite ];
+
+  outputs = [ "out" "man" ];
+
+  postInstall = ''
+    export OUT_DIR=$(mktemp -d)
+
+    # Generate the man pages
+    cargo run --bin flawz-mangen
+    installManPage $OUT_DIR/flawz.1
+
+    # Generate shell completions
+    cargo run --bin flawz-completions
+    installShellCompletion \
+      --bash $OUT_DIR/flawz.bash \
+      --fish $OUT_DIR/flawz.fish \
+      --zsh $OUT_DIR/_flawz
+
+    # Clean up temporary directory
+    rm -rf $OUT_DIR
+  '';
+
+  meta = with lib; {
+    description = "A Terminal UI for browsing CVEs";
+    longDescription = ''
+      flawz is a Terminal User Interface (TUI) for browsing the security vulnerabilities (also known as CVEs).
```

Comment:
Does this really add much to the short description?

Reply:
no, i just thought that it's required. i'll delete it thx